ddl/ddl_test.go

```diff
@@ -61,6 +62,24 @@ func (d *ddl) generalWorker() *worker {
 	return d.workers[generalWorker]
 }
 
+// restartWorkers is like the function fo d.start. But it won't initialize the "workers" and create a new worker.
```

Comment:
s/fo/for?

Reply:
What do you mean by "is like"?

Reply:
The operation of these two functions is similar.

Reply:
For this case, either is like the function d.start() or is similar with the function d.start()